src/loaders/compute_tools/tool_common.py

```diff
@@ -218,7 +237,7 @@ def _get_data_ids(self):
             data_ids = all_data_ids
         return list(set(data_ids))
 
-    def parallel_single_execution(self, tool_callable: Callable[[str, Path, Path, bool], None], unzip=False):
+    def parallel_single_execution(self, tool_callable: Callable[[str, Path, Path, bool, str], None], unzip=False):
```

Comment:
Putting the data ID at the end after debug seems weird. I'd put it right next to the genome_id / tool identifier

Reply:
Updated!

Reply:
:+1: